internal/handlers/common/msg_listcommands.go

```diff
@@ -205,9 +209,18 @@ func MsgListCommands(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
 	cmdList := must.NotFail(types.NewDocument())
 	names := maps.Keys(Commands)
 	sort.Strings(names)
+
+	ignoreLowercaseCmds := map[string]bool{
```

Comment:
Do we need a separate map that we should keep in sync with `Commands`? Maybe we could skip commands with an empty Help field?

Reply:
That's definitely easier to maintain.

Reply:
So, should I remove the map and skip empty `Help` fields instead? And add something similar to [this](https://github.com/FerretDB/FerretDB/blob/main/integration/commands_diagnostic_test.go#L255) for all three commands?